ballerina/api_inspectionplan_srv/Module.md

```diff
@@ -0,0 +1,80 @@
+## Overview
+
+[S/4HANA](https://www.sap.com/india/products/erp/s4hana.html) is a robust enterprise resource planning (ERP) solution,
+designed for large-scale enterprises by SAP SE.
+
+The `ballerinax/sap.s4hana.api_inspectionlot_srv` package offers APIs for seamless integration with the [Inspection Lot API v1.2.0](https://api.sap.com/api/API_INFORECORD_PROCESS_SRV/overview).In the SAP system, you process quality inspections on the basis of inspection lots. The data relating to a quality inspection is grouped into several entities. With this service it is possible, for example, to update inspection lot header data or to create inspection results and usage decisions.
```

Comment:
Is the link to API documentation correct?

Reply:
Please find fix in [Update docs.json in two modules to fix automated document generation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/10c211e4570ea2d2501780ca1029d936b69f092b)